src/core_modules/capture-core/components/WidgetTwoEventWorkspace/OverflowMenu/Modal/UnlinkModal.tsx

```diff
@@ -63,20 +62,20 @@ export const UnlinkModal = ({
                 {i18n.t('Unlink event')}
             </ModalTitle>
             <ModalContent>
-                <p>{i18n.t('Are you sure you want to remove the link between these events? This action removes the link itself, but the linked event will remain.')}</p>
+                <p>{i18n.t('Are you sure you want to remove the link between these events? This action removes the link itself, but the linked event will remain.') as string}</p>
             </ModalContent>
             <ModalActions>
                 <ButtonStrip end>
                     <Button onClick={() => setOpenModal(false)} secondary>
-                        {i18n.t('No, cancel')}
+                        {i18n.t('No, cancel') as string}
                     </Button>
                     <Button
                         destructive
                         onClick={() => mutation.mutate()}
                         disabled={mutation.isLoading}
                         dataTest="event-overflow-unlink-event-confirm"
                     >
-                        {i18n.t('Yes, unlink event')}
+                        {i18n.t('Yes, unlink event') as string}
```

Comment:
Remove the `as string`. Remember to never add `as string` after the ` i18n.t(`) function